public/js/graph/react_graph.js

```diff
@@ -159,9 +157,8 @@ var Graph = React.createClass({
             horizontalPanFactor: 0,
             verticalPanFactor: 0,
             mouseDown: false,
-            modalIsOpen: false,
-            courseId: 'csc108H1'
-            };
+            courseId: ''
```

Comment:
Yes you're right, I don't call it when opening the modal, I use the courseId straight from the info box. I will fix this as well.